### PR TITLE
adding swg button

### PR DESCRIPTION
**Pull request by prateekbh**
- Adds SwG button in the dialog

### CODE REVIEW

File: app/views/article-amp.html [hunk 1]
```diff
@@ -447,6 +447,8 @@ <h1 class="related">Related articles</h1>
         <a subscriptions-action="subscribe" subscriptions-display="TRUE">Subscribe</a>
         or
         <a subscriptions-action="login" subscriptions-display="TRUE">Login</a>
+        <div subscriptions-action="subscribe" subscriptions-display="TRUE"
+          subscriptions-service="subscribe.google.com" subscriptions-decorate>Subscribe</div>
```
Comment (dvoytenko) [inline app/views/article-amp.html]: `subscription-decorate` is default, right? Actually, we need to remove it in either case from here for now - Scenic must always be valid, so we can't use `decorate` until validator is updated everywhere.

File: app/views/article-amp.html [hunk 2]
```diff
@@ -447,6 +447,8 @@ <h1 class="related">Related articles</h1>
         <a subscriptions-action="subscribe" subscriptions-display="TRUE">Subscribe</a>
         or
         <a subscriptions-action="login" subscriptions-display="TRUE">Login</a>
+        <div subscriptions-action="subscribe" subscriptions-display="TRUE"
```
Comment (dvoytenko) [inline app/views/article-amp.html]: Why do we need `subscriptions-display` at all in this case?

Comment (prateekbh) [reply app/views/article-amp.html]: because otherwise we wont run eval without `subscriptions-display` and so it'll be invisible.

Comment (dvoytenko) [reply app/views/article-amp.html]: Yeah. Right. Would be good to find some other ideas around that in the future.

**Pull request merged**